lib/tasks/gemcutter.rake

```diff
@@ -79,4 +79,24 @@ namespace :gemcutter do
       puts "Done."
     end
   end
+
+  namespace :rubygems_version do
+    desc "Backfill gem versions with rubygems_version."
+    task backfill: :environment do
+      without_rubygems_version = Version.where(rubygems_version: nil)
+      mod = ENV['shard']
```

Comment:
I added this just cause rake task to [backfill metadata](https://github.com/rubygems/rubygems.org/pull/1276/files#diff-057c0b82f700094d44295e5d85fc7a97R67) had it. Let me know if it is unnecessary.